docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,171 @@
+# Creating new servers with virtual GPU support
+
+In {{brand}}, you can instantiate virtual machines --- or simply
+_servers_ --- with virtual GPU (vGPU) support. For that, you use the
+`openstack` CLI utility.
+
+## Prerequisites
+
+Although not strictly necessary, it is certainly helpful to have some
+familiarity with [creating servers in {{brand}}](new-server.md),
+especially with the OpenStack CLI.
+
+## Instantiating a vGPU server
+
+Before you begin, please make sure to source [your `openstack` RC file
+first](../../getting-started/enable-openstack-cli.md).
+
+As we discussed in the [guide on creating
+servers](/howto/openstack/nova/new-server), an `openstack` command for
+instantiating a new server looks like this:
+
+```console
+openstack server create \
+    --flavor $FLAVOR_ID \
+    --image $IMAGE_NAME \
+    --boot-from-volume $VOL_SIZE \
+    --network $NETWORK_NAME \
+    --security-group $SEC_GROUP_NAME \
+    --key-name $KEY_NAME \
+    --wait \
+    $SERVER_NAME
+```
+
+For servers with vGPU support, the only acceptable value for variable
+`IMAGE_NAME` is `Ubuntu 22.04 NVGRID x86_64`. Additionally, the value
+for variable `FLAVOR_ID` should be the ID of a flavor prefixed by
+"`g.`". To see all such flavors, along with the corresponding IDs, you
```

Comment:
Cross-reference please, and also you can drop the double quotes and the extra full-stop:
```suggestion
for variable `FLAVOR_ID` should be the ID of a flavor [prefixed by
`g`](../../../reference/flavors/index.md#compute-tiers). 
To see all such flavors, along with the corresponding IDs, you
```
But a different thought: why are we referring to flavors by their UUID here, when we implore users to refer to images by name and never use UUIDs? Why don't we suggest to use flavor names?

Reply:
Uhm, yes, names for flavors -- just like names for images. I have adapted the prose accordingly.